lib/foreman_discovery/engine.rb

```diff
@@ -107,6 +103,7 @@ class Engine < ::Rails::Engine
           :view_organizations,
           :view_locations,
           :view_hosts,
+          :view_operatingsystems,
```

Comment:
Yeah, it does not add, you need to add manually. Can you add the migration above? There are no security concerns about this one, we can simply add it. If the permission is already present, we can add second instance, no big deal.